gql_schema_codegen/block/block.py

```diff
@@ -59,15 +70,28 @@ def heading_file_line(self):
         )
 
         if not self.implements:
+            # check if we have an interface implementing another interface
+            deps = get_interface_dependencies()
+            if display_name in deps:
+                siblings = inheritanceTree.get(deps[display_name], set())
+                siblings.add(display_name)
+                inheritanceTree[deps[display_name]] = siblings
```

Comment:
```suggestion
                inheritanceTree.setdefault(deps[display_name], set()).add(display_name)
```

https://docs.python.org/3.11/library/stdtypes.html#dict.setdefault

or if you do `defaultdict` as per above:

```suggestion
                inheritanceTree[deps[display_name]].add(display_name)
```